packages/arb-token-bridge-ui/src/util/L2ApprovalUtils.ts

```diff
@@ -66,10 +67,21 @@ const L2ApproveTokens: { [chainId: number]: RequireL2ApproveToken[] } = {
   ]
 }
 
-export function tokenRequiresApprovalOnL2(
+export async function tokenRequiresApprovalOnL2(
```

Comment:
Maybe we could have an object as a param for both `tokenRequiresApprovalOnL2` and `xErc20RequiresApprovalOnChildChain`?